foreshadow/tests/test_core/test_gui.py

```diff
@@ -0,0 +1,86 @@
+"""Tests for the gui DTO objects."""
+import pytest
+
+
+@pytest.mark.parametrize("event_data", [{}, {"invalid_key": "invalid_value"}])
+def test_gui_event(event_data):
+    """Test that the gui event automatically adds timestamp and throws errors.
+
+    Args:
+        event_data: the data to load
+
+    """
+    from foreshadow.core.gui import GuiEvent
+    from datetime import datetime
+
+    schema = GuiEvent()
+    now = datetime.now().strftime("%Y%m%dT%H:%M:%S:%f")
+    data = schema.load(event_data).data
+    now_time = now[now.find("T") + 1 :]  # noqa: E203 black/flake8 issue
```

Comment:
There are two options to avoid the noqa, did some research and bumped a few issues :)

1. Make the line two instead
```python
some_meaningful_name = now.find("T") + 1 
now_time = now[some_meaningful_name:]
```
2. Keep the noqa as is

Reply:
It's a known issue: https://github.com/PyCQA/pycodestyle/issues/373

Reply:
I see